oteapi_asmod/strategies/parse.py

```diff
@@ -63,24 +68,26 @@ def parse(self, session: "Optional[Dict[str, Any]]" = None) -> "Dict[str, Any]":
             session: A session-specific dictionary context.
 
         Returns:
-            Dictionary of key/value-pairs to be stored in the sessions-specific
-            dictionary context.
+            ase.Atoms object in cache.
 
         """
         model = AtomisticParseDataModel(
-            **self.resource_config.configuration, extra=Extra.ignore
+            **self.parse_config.configuration, extra=Extra.ignore
         )
 
-        downloader = create_download_strategy(self.resource_config)
+        downloader = create_strategy("download", self.parse_config)
         output = downloader.get()
-        cache = DataCache(self.resource_config.configuration)
+        cache = DataCache(self.parse_config.configuration)
```

Comment:
The `self.parse_config.configuration` is to configure the `AtomisticParseDataModel`. If you wan't to allow to configure the data cache, you can add a `datacache_config` field to AtomisticParseDataModel and provide that as the argument to `DataCache()`.

```suggestion
        cache = DataCache()
```